scripts/update/Updater.php

```diff
@@ -259,5 +241,11 @@ public function update($initialVersion) {
         }
 
         $this->skip('7.6.0', '9.0.1');
+
+        if ($this->isVersion('9.0.1')) {
+            $this->getServiceManager()->unregister('taoDeliveryRdf/AssemblerService');
+
+            $this->setVersion('10.0.0');
```

Comment:
I guess that is just refactoring, nothing new or breaking here, it works without any changes